### PR TITLE
Fix mission control parser issue 58

### LINKED ISSUE #58: Fix Drive MissionControlParser
should parse the following format:
```
{
"heartbeat_count": null,
"is_operational": 1,
"wheel_shift": 0,
"drive_mode": "S",
"speed": 0,
"angle": 0
}
```

### PR DESCRIPTION
**Pull request by FluffyFTW**
fixed drive ParseMissionControlData
resolves #58

### CODE REVIEW

**Review by acarrou — APPROVED** [review]
Yeet indeed

File: subsystem-drive/implementations/mission-control-handler.hpp [hunk 1]
```diff
@@ -5,12 +5,24 @@
...
+    /*
+        {
+        "heartbeat_count": null,
+        "is_operational": 1,
+        "wheel_shift": 0,
+        "drive_mode": "S",
+        "speed": 0,
+        "angle": 0
+        }
+    */
```
Comment (naterpotatoers) [inline subsystem-drive/implementations/mission-control-handler.hpp:17]: Get rid of this

Comment (naterpotatoers) [inline subsystem-drive/implementations/mission-control-handler.hpp:17]:
```suggestion
```

**Review by naterpotatoers — CHANGES_REQUESTED** [review]
Get this done Gene!

**Review by naterpotatoers — APPROVED** [review]
Looks good :)

**Pull request merged**